lib/stink_bomb/configuration.rb

```diff
@@ -7,21 +7,11 @@ def initialize(options = {})
     end
 
     def raise=(error_class)
-      remove_bomb(RaiseBomb)
-      if error_class == true
-        add_bomb(RaiseBomb.new)
-      elsif error_class
-        add_bomb(RaiseBomb.new(error_class: error_class))
-      end
+      add_bomb(RaiseBomb, error_class)
```

Comment:
yeah, but it's not really a singleton either. Another option would be to add a little duplication like:

```ruby
def raise=(error_class)
  remove_bomb(RaiseBomb)
  add_bomb(RaiseBomb, error_class) if error_class
end

def add_bomb(bomb_class, bomb_arg)
  if bomb_arg == true
    bombs << bomb_class.new
  else
    bombs << bomb_class.new(bomb_arg)
  end
end
```